datadog/fwprovider/resource_datadog_software_catalog.go

```diff
@@ -41,7 +42,8 @@ type Included struct {
 }
 
 type IncludedAttributes struct {
-	Schema *Entity `json:"schema"`
+	Schema    *Entity `json:"schema"`
```

Comment:
do we still need `Schema` ?

Reply:
I thought so, because it's that's used in updating the entity (in `resourceEntityUpsert()`), which I thought is separate from the read. Would my understanding be correct?